src/components/FeaturedProjectCard.js

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable react/jsx-closing-bracket-location */
+import React from 'react';
+import styled from 'styled-components/macro';
+import { ProjectTags, ceruleanBlue, TagWrapper, SectionHeader } from 'Styles';
+import FeaturedProject from './FeaturedProject.json';
+
+const FeaturedProjectCard = () => {
+  const color = ceruleanBlue;
+  const GITHUB_NAME = 'joeohm';
+
+  return (
+    <>
+      <SectionHeader color={color}>FEATURED PROJECTS</SectionHeader>
+      <FeaturedProjectWrapper>
+        {FeaturedProject.map((project) => {
+          return (
+            <ProjectCard
+              key={project.title}
+              href={`https://github.com/${GITHUB_NAME}/${project['repo-name']}`}
+              target="_blank"
+            >
+              <ThumbnailWrapper url={project.image}>
+                <ThumbnailTitle>{project.title.toUpperCase()}</ThumbnailTitle>
+              </ThumbnailWrapper>
+              <ProjectInfoHeader color={color}>
+                {project['title-description'].toUpperCase()}
+              </ProjectInfoHeader>
+              <ProjectInfo>{project['project-description']}</ProjectInfo>
+              <TagWrapper>
+                {project.tags.map((tag) => (
+                  <ProjectTags key={tag}>{tag}</ProjectTags>
+                ))}
+              </TagWrapper>
+            </ProjectCard>
+          );
+        })}
+      </FeaturedProjectWrapper>
+    </>
+  );
+};
+
+export default FeaturedProjectCard;
+
+// const SectionHeader = styled.h2`
+//   font-size: 50px;
+//   line-height: 1;
+//   padding: 10px 15px;
+//   margin-bottom: 100px;
+//   font-family: 'Montserrat Bold';
+//   background-color: ${(props) => props.color};
+//   color: white;
+
+//   @media (max-width: 800px) {
+//     font-size: 22px;
+//     line-height: 1;
+//     padding: 10px 15px;
+//     margin-bottom: 40px;
+//   }
+// `;
+
+const FeaturedProjectWrapper = styled.div`
+  text-align: left;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 50px;
```

Comment:
If you use Percent insteas of pixels you dont have to specify as much in the media queries and the code becomes simpler